ezcontent_publish.module

```diff
@@ -102,20 +102,20 @@ function ezcontent_publish_form_shortcut_set_customize_form_alter(&$form, $form_
  */
 function ezcontent_publish_form_alter(&$form, $form_state, $form_id) {
   $currentUser = \Drupal::currentUser();
-  if (!in_array('administrator', $currentUser->getRoles())) {
-    $form['shortcuts']['links']['#empty'] = t('No shortcuts available. <a href=":link">Add a shortcut</a>', [':link' => Url::fromRoute('ezcontent_publish.shortcut.link_add', ['shortcut_set' => 'shortcut_set_' . $currentUser->getUsername() . '_' . $currentUser->id()])->toString()]);
-    if ($form_id === 'shortcut_confirm_delete' || $form_id === 'shortcut_shortcut_set_' . $currentUser->getUsername() . '_' . $currentUser->id() . '_form' || $form_id === 'shortcut_shortcut_set_' . $currentUser->getUsername() . '_'  . $currentUser->id() . '_add_form') {
-      $form['actions']['submit']['#submit'][] = 'ezcontent_publish_shortcut_form_submit';
-    }
+  $form['shortcuts']['links']['#empty'] = t('No shortcuts available. <a href=":link">Add a shortcut</a>', [':link' => Url::fromRoute('ezcontent_publish.shortcut.link_add', ['shortcut_set' => 'shortcut_set_' . $currentUser->getUsername() . '_' . $currentUser->id()])->toString()]);
+  if ($form_id === 'shortcut_default_add_form' || $form_id === 'shortcut_set_customize_form' || $form_id === 'shortcut_default_form' || $form_id === 'shortcut_confirm_delete' || $form_id === 'shortcut_shortcut_set_' . $currentUser->getUsername() . '_' . $currentUser->id() . '_form' || $form_id === 'shortcut_shortcut_set_' . $currentUser->getUsername() . '_'  . $currentUser->id() . '_add_form') {
```

Comment:
Please declare an array of form_ids and then do an in_array check, something like this:

`$form_ids = ['form_id_1', 'form_id_2', ..., 'form_id_n'];`
`if (in_array($form_id, $form_ids))`

This would make it more readable.